code/core/system/proxy-velodyne16/include/velodyne16Decoder.h

```diff
@@ -16,73 +16,73 @@
-#include "opendavinci/generated/odcore/data/SharedPointCloud.h"
+#include <opendavinci/odcore/io/StringListener.h>
 
 namespace opendlv {
 namespace core {
 namespace system {
 namespace proxy {
-    using namespace odcore::wrapper;
+using namespace odcore::wrapper;
 
-    // This class will handle bytes received via a UDP socket.
-    class velodyne16Decoder : public odcore::io::StringListener {
-    private:
-        /**
+// This class will handle bytes received via a UDP socket.
+class velodyne16Decoder : public odcore::io::StringListener {
+   private:
+    /**
          * "Forbidden" copy constructor. Goal: The compiler should warn
          * already at compile time for unwanted bugs caused by any misuse
          * of the copy constructor.
          */
-        velodyne16Decoder(const velodyne16Decoder &);
+    velodyne16Decoder(const velodyne16Decoder &);
 
-        /**
+    /**
          * "Forbidden" assignment operator. Goal: The compiler should warn
          * already at compile time for unwanted bugs caused by any misuse
          * of the assignment operator.
          */
-        velodyne16Decoder& operator=(const velodyne16Decoder &);
+    velodyne16Decoder &operator=(const velodyne16Decoder &);
 
-    public:
-        /**
+   public:
+    /**
          * Constructor.
          */
-        velodyne16Decoder(std::shared_ptr<SharedMemory>,odcore::io::conference::ContainerConference&);
+    velodyne16Decoder(const std::shared_ptr< SharedMemory >, odcore::io::conference::ContainerConference &, const string &);
 
-        virtual ~velodyne16Decoder();
-        
-        void sendSPC(const float &oldAzimuth, const float &newAzimuth);
-        
-        virtual void nextString(const std::string &s);
-    
-    private:
-        const uint32_t MAX_POINT_SIZE=30000;  //the maximum number of points per frame. This upper bound should be set as low as possible, as it affects the shared memory size and thus the frame updating speed.
-        const uint32_t SIZE_PER_COMPONENT = sizeof(float);
-        const uint8_t NUMBER_OF_COMPONENTS_PER_POINT = 4; // How many components do we have per vector?
-        const uint32_t SIZE = MAX_POINT_SIZE * NUMBER_OF_COMPONENTS_PER_POINT * SIZE_PER_COMPONENT; // What is the total size of the shared memory?    
-        const float PI=3.14159;
-        
-        uint32_t pointIndex;
-        uint32_t startID;
-        float previousAzimuth;
-        float deltaAzimuth;
-        float distance;
-        std::shared_ptr<SharedMemory> VelodyneSharedMemory;//shared memory for the shared point cloud
-        float* segment;//temporary memory for transferring data of each frame to the shared memory
-        odcore::io::conference::ContainerConference& velodyneFrame;
-        odcore::data::SharedPointCloud spc;//shared point cloud
-        float vertCorrection[16];  //Vertal angle of each sensor beam  
-    };
+    virtual ~velodyne16Decoder();
+
+    void sendSPC(const float &oldAzimuth, const float &newAzimuth);
```

Comment:
Please use more explanatory names and reduce this method's visibility as much as possible (i.e. from public to private as it is used only internally, right?)